fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/BardLoggingFilter.java

```diff
@@ -83,6 +84,7 @@
     @Override
     public void filter(ContainerRequestContext request) throws IOException {
 
+        RequestLog.addIdPrefix(request.getHeaders().getFirst(X_REQUEST_ID_HEADER));
```

Comment:
And I would say that min-length doesn't matter, and 200 chars for max length seems sane. Stripping any unallowed character (and then truncating) is probably the most permissive approach too